app/views/shelters/edit.html.erb

```diff
@@ -1,18 +1,18 @@
-<%= form_tag "/shelters/#{@shelter.id}", method: :patch do %>
+<%= form_tag "/shelters/#{@shelter.id}", method: :put do %>
   <%= label_tag :name %>
-  <%= text_field_tag :name %><br/>
+  <%= text_field_tag :name, @shelter.name%><br/>
```

Comment:
`@shelter.name` will pre-populate the form with the shelter's previously recorded name, same with the text_field_tags below